protocol-designer/src/steplist/substepTimeline.ts

```diff
@@ -1,78 +1,77 @@
-// @flow
 import last from 'lodash/last'
 import pick from 'lodash/pick'
 import {
   getWellsForTips,
   getNextRobotStateAndWarningsSingleCommand,
 } from '@opentrons/step-generation'
-
-import type { Command } from '@opentrons/shared-data/protocol/flowTypes/schemaV6'
-import type { Channels } from '@opentrons/components'
+import { Command } from '@opentrons/shared-data/protocol/types/schemaV6'
+import { Channels } from '@opentrons/components'
 import type {
   CommandCreatorError,
   CommandsAndWarnings,
   CurriedCommandCreator,
   InvariantContext,
   RobotState,
 } from '@opentrons/step-generation'
-import type { SubstepTimelineFrame, SourceDestData, TipLocation } from './types'
+import { SubstepTimelineFrame, SourceDestData, TipLocation } from './types'
 
 /** Return last picked up tip in the specified commands, if any */
-export function _getNewActiveTips(commands: Array<Command>): ?TipLocation {
-  const lastNewTipCommand: ?Command = last(
+export function _getNewActiveTips(
+  commands: Command[]
+): TipLocation | null | undefined {
+  const lastNewTipCommand: Command | null | undefined = last(
     commands.filter(c => c.command === 'pickUpTip')
   )
   const newTipParams =
     (lastNewTipCommand != null &&
       lastNewTipCommand.command === 'pickUpTip' &&
       lastNewTipCommand.params) ||
     null
-
   return newTipParams
 }
 
-const _createNextTimelineFrame = ({
-  command,
-  index,
-  nextFrame,
-  volume,
-  wellInfo,
-}: {
-  command: Command,
-  index: number,
-  nextFrame: CommandsAndWarnings,
-  volume: number,
-  wellInfo: SourceDestData,
-}) => {
+const _createNextTimelineFrame = (args: {
+  command: Command
+  index: number
+  nextFrame: CommandsAndWarnings
+  volume: number
+  wellInfo: SourceDestData
+}): Partial<{
+  command: Command
+  index: number
+  nextFrame: CommandsAndWarnings
+  volume: number
+  wellInfo: SourceDestData
+}> => {
   // TODO(IL, 2020-02-24): is there a cleaner way to create newTimelineFrame
   // and keep Flow happy about computed properties?
```

Comment:
If this comment still applies in TS, then
```suggestion
  // and keep TS happy about computed properties?
```